pkg/sgpp/constants.go

```diff
@@ -3,8 +3,12 @@ package sgpp
 type EnforcingMode string
 
 const (
-	EnforcingModeStrict   EnforcingMode = "strict"
-	EnforcingModeStandard EnforcingMode = "standard"
+	EnforcingModeStrict               EnforcingMode = "strict"
+	EnforcingModeStandard             EnforcingMode = "standard"
+	VpcCNINodeEventActionForTrunk     string        = "NeedTrunk"
+	TrunkEventNote                    string        = "vpc.amazonaws.com/has-trunk-attached=false"
```

Comment:
Are we not generating event for `vpc.amazonaws.com/has-trunk-attached=true`? I believe we were trying to address that for nodes that come up with a trunk interface attached.

Reply:
We do check `c.dataStore.GetTrunkENI() == "" || !hasTrunkLabel` to send an event if a node has no trunk or has no label. We don't check label value and only check label key that means we check for `vpc.amazonaws.com/has-trunk-attached` not the boolean value.

```
if labels != nil {
		_, hasTrunkLabel = labels[trunkInterfaceLabel]
	}
```